README.rst

```diff
@@ -181,6 +182,9 @@ Starting the app
   # start the app
   $ docker-compose up -d db app
 
+  # Apply the migrations
+  $ docker-compose run manage migrate
```

Comment:
So actually I think it's working as expected now, as in I don't think we need to manually run the migrations. They were failing to run previously because of a dependency issue, but since that got resolved I believe the migrations are being run in the setup.

Reply:
yes. But I added the command to README just in case if somebody makes a change to django model and might be wondering how to run migrations in docker. I can add "(optional)"  to the instruction hinting it isn't a necessary step and also add makemigrations command. Or if you think such an instance of other users making modifications to django models doesn't happen I can get rid of it.

Reply:
Honestly I think either of the things you suggested would be fine

Reply:
If we keep it, maybe the instructions could specify something like: 
```
# If you make changes to the models, create and apply a migration
$ docker-compose run manage makemigrations
$ docker-compose run manage migrate
```